custom_components/clientraw/sensor.py

```diff
@@ -226,7 +255,7 @@ def try_again(err: str):
                 new_state = round(speed, 2)
 
             elif dev.type == 'wind_gust':
-                gust = float(self.data[2])
+                gust = float(self.data[133])
```

Comment:
Why not use index 71?

Reply:
I think 71 is max gust for the day? Usually a weather briefing will talk about current winds and current gusts - I might change the name to be wind_gust_hour and add 71 as wind_gust_day ...?

Reply:
Please do so

Reply:
Ok these now added